File: dailyQuestion/2020-08/08-05/python/solution.py
def findRepeatNumber(nums: [int]) -> int:
    tmp = {}
    for num in nums :
        if num in tmp :
            return num
        else :
            tmp[num] = True
    return

if __name__ == "__main__" :
    nums = [2, 3, 1, 0, 2, 5, 3]
    result = findRepeatNumber(nums)
    print(result)

File: dailyQuestion/2020-08/08-05/golang/solution.go
package main

import "fmt"

func findRepeatNumber(nums []int) int {
	tmp := map[int]bool{}
	for _, item := range nums {
		if tmp[item] {
			return item
		} else {
			tmp[item] = true
		}
	}
	return 0
}

func main() {
	nums := []int{2, 3, 1, 0, 2, 5, 3}
	result := findRepeatNumber(nums)
	fmt.Println(result)
}
